packages/stacking/README.md

```diff
@@ -94,13 +94,27 @@ const privateKey = 'd48f215481c16cbe6426f8e557df9b78895661971d71735126545abddcd5
 // block height at which to stack
 const burnBlockHeight = 2000;
 
+// signer key
+const signerPrivateKey = randomPrivKey();
+const signerKey = getPublicKeyFromPrivate(signerPrivateKey.data);
+
 // Refer to initialization section to create client instance
+const signerSignature = client.signPoxSignature({
+  topic: 'stack-stx',
+  rewardCycle: 132, // current reward cycle
```

Comment:
Should we include a snippet of code around here for the best way to grab the reward cycle?

Reply:
Yeah, this one is weird. It should be the current reward-cycle, if I read the pox-4 correctly. We could also default to current and do a fetch call to get the current (requires async, that's why I left it out for now).